src/detours.cpp

```diff
@@ -76,7 +76,10 @@ namespace rhost {
                 hWndTitleMap.find(hWnd) == hWndTitleMap.end() && 
                 SUCCEEDED(StringCchLengthA(lpWindowName, STRSAFE_MAX_CCH, &len)) && len > 0) {
 
-                hWndTitleMap[hWnd] = std::string(lpWindowName);
+                std::string windowTitle = std::string(lpWindowName);
+                std::wstring_convert<std::codecvt_utf8<char>, char> strConverter;
```

Comment:
What is the correct way to convert this? I am not familiar with these conversions.